### ISSUE TITLE
Diffuse lupus nephritis

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6835]
**Preferred term label:**
Diffuse lupus nephritis

**Synonyms**
Class IV lupus nephritis

**Definition (free text, please give PubMed ID)**
A lupus nephritis subclass with active or inactive diffuse, segmental or global endocapillary or extracapillary glomerulonephritis involving ≥50% of all glomeruli, typically with diffuse subendothelial immune deposits, with or without mesangial alterations.
PMID: 31974366

**Parent term (use hpo.jax.org/app)**
Lupus nephritis (newly proposed term)

**Related terms** 
Mesangial hypercellularity HP:0012574
Mesangial Immune complex deposition HP:0033295
Systemic lupus erythematosus HP:0002725

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM #152700, ORPHA:536

**Your nano-attribution (ORCID)**
OMIM #152700, ORPHA:536

**Comment by pnrobinson** [comment]
added
